networking_oneview/ml2/drivers/oneview/neutron_oneview_client.py

```diff
@@ -398,8 +350,8 @@ def create(self, session, port_dict):
         server_hardware = (
             common.server_hardware_from_local_link_information_list(
                 self.oneview_client, local_link_information_list))
-        server_profile = self.server_profile_from_server_hardware(
-            server_hardware
+        server_profile = common.server_profile_from_server_hardware(
+            self.oneview_client, server_hardware
```

Comment:
I don't think we need to pass the oneview_client, we could have get_oneview_client within the method.